metrics.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // Metrics is a wrapper interface for go-metrics
-// support Counter, Gauge Histogram
+// support Counter, Gauge, Histogram, EWMA
```

Comment:
or, maybe it is worth some doc link for EWMA? Thanks!

Reply:
https://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average.

I think it's should be simply in comments :)